extension/src/experiments/model/quickPick.ts

```diff
@@ -146,7 +142,7 @@ export const pickExperiment = (
   experiments: Experiment[],
   firstThreeColumnOrder: string[],
   title: Title = Title.SELECT_EXPERIMENT
-): Thenable<ExperimentDetails | undefined> =>
+): Thenable<string | undefined> =>
```

Comment:
[F] As there is now no difference between id/name (if there is one) we don't need the `ExperimentDetails` type.